README.md

```diff
@@ -1,19 +1,25 @@
 # triton-service-groups
 
+Triton Service Groups provide monitoring for Triton applications, automatically scaling to meet capacity needs and maintain performance at the lowest possible cost. Manage applications using the TSG API or with Terraform to scale resources on Triton including VMs and containers. 
```

Comment:
> automatically scaling to meet capacity needs

I think this is a misunderstanding. The TSG emphatically does not provide any automatic scaling. 

It _does_ give users an easy button they can use to change instance counts.

Reply:
Oh okay, 100% did not get that. That changes this introduction.